changelog/8.17.6.asciidoc

```diff
@@ -0,0 +1,33 @@
+// begin 8.17.6 relnotes
+
+[[release-notes-8.17.6]]
+==  8.17.6
+
+Review important information about the  8.17.6 release.
+
+
+
+
+
+
+
+
+
+[discrete]
+[[new-features-8.17.6]]
+=== New features
+
+The 8.17.6 release adds the following new and notable features.
+
+
+elastic-agent::
+
+* Set Collectors Fullnameoverride For Edot Kube-Stack Values. {elastic-agent-pull}https://github.com/elastic/elastic-agent/pull/7754[#https://github.com/elastic/elastic-agent/pull/7754] {elastic-agent-issue}https://github.com/elastic/elastic-agent/issues/7381[#https://github.com/elastic/elastic-agent/issues/7381]
```

Comment:
```suggestion
* Use `fullnameOverride` to set the fully qualified application names in the EDOT Kube-Stack Helm chart. {elastic-agent-pull}https://github.com/elastic/elastic-agent/pull/7754[#https://github.com/elastic/elastic-agent/pull/7754] {elastic-agent-issue}https://github.com/elastic/elastic-agent/issues/7381[#https://github.com/elastic/elastic-agent/issues/7381]
```

I had to read the linked issue and PR and then do some additional searching to figure out what this actually did.

Reply:
I add this change into the docs PR https://github.com/elastic/ingest-docs/pull/1789